### DISCUSSION TITLE
Allow more customization file types

### DISCUSSION THREAD

**Discussion by Seb33300** [PrestaShop/PrestaShop#40143] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Describe the solution you'd like

When creating a customizable product, in the `Options` tab of the product page in the admin panel, we can enable file upload for the customer in the customization section.

Unfortunately, this will only allow upload of image file types which will satisfy only a limited number of use cases.
Most of the time we will also want to allow to upload PDF, Word, Zip file types or some other specific types like Photoshop or AI file types.

I think this option should be improved to let us to choose what types of files we want to allow for each file input.

![image](https://user-images.githubusercontent.com/915273/162884561-c7e7eefd-f336-4fc1-b5a5-b415910c3021.png)

**Comment by Hlavtox** [comment]
I did this modification many times, it isn't even that hard to do. :-)

**Comment by Seb33300** [comment]
@Hlavtox Can you please share how you did it? Or maybe you can work on a pull request?

**Comment by marwachelly** [comment]
Hello @Seb33300 ,

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. 

@Hlavtox thanks for your quick reply and we're waiting for your PR :rocket: 

Thanks,

**Comment by caygri** [comment]
Hello, how can I add extra product type?

**Comment by Seb33300** [comment]
I used this module to solve my issue:
https://addons.prestashop.com/en/front-office-features-prestashop-modules/88383-upload-any-file-type-in-product-customization.html
